apps/docs-snippets/src/guide/wallets/connectors.test.ts

```diff
@@ -0,0 +1,221 @@
+  metadata = {
+    install: metadata.install,
+    image: metadata.image,
+  };
+  // #endregion fuel-connector-metadata
+
+  private eventsConnectorEvents(): void {
+    // #region fuel-connector-events-accounts
+    const accounts: Array<string> = ['0x1234567890abcdef'];
+
+    this.emit(this.events.accounts, accounts);
+    // #endregion fuel-connector-events-accounts
+
+    // #region fuel-connector-events-connection
+    const connection: boolean = true;
+
+    this.emit(this.events.connection, connection);
+    // #endregion fuel-connector-events-connection
+
+    // #region fuel-connector-events-connectors
+    const connectors: Array<FuelConnector> = [new MyWalletConnector()];
+
+    this.emit(this.events.connectors, connectors);
+    // #endregion fuel-connector-events-connectors
+
+    // #region fuel-connector-events-currentConnector
+    const currentConnector: FuelConnector = new MyWalletConnector();
+
+    this.emit(this.events.currentConnector, currentConnector);
+    // #endregion fuel-connector-events-currentConnector
+
+    // #region fuel-connector-events-currentAccount
+    const currentAccount: string = '0x1234567890abcdef';
+
+    this.emit(this.events.currentAccount, currentAccount);
+    // #endregion fuel-connector-events-currentAccount
+
+    // #region fuel-connector-events-networks
+    // #import { Network };
+
+    const network: Network = {
+      chainId: 1,
+      url: 'https://example.com/rpc',
+    };
+
+    this.emit(this.events.networks, network);
+    // #endregion fuel-connector-events-networks
+
+    // #region fuel-connector-events-currentNetwork
+    // #import { Network };
+
+    const currentNetwork: Network = {
+      chainId: 1,
+      url: 'https://example.com/rpc',
+    };
+
+    this.emit(this.events.currentNetwork, currentNetwork);
+    // #endregion fuel-connector-events-currentNetwork
+
+    // #region fuel-connector-events-assets
+    // #import { Asset };
+
+    const assets: Array<Asset> = [
+      {
+        name: 'Ethereum',
+        symbol: 'ETH',
+        icon: 'https://cdn.fuel.network/assets/eth.svg',
+        networks: [
+          {
+            type: 'ethereum',
+            chainId: 11155111,
+            decimals: 18,
+          },
+        ],
+      },
+    ];
+
+    this.emit(this.events.assets, assets);
+    // #endregion fuel-connector-events-assets
+
+    // #region fuel-connector-events-abis
+    const abis: Array<FuelABI> = [
+      {
+        encoding: '1',
+        types: [],
+        loggedTypes: [],
+        functions: [],
+        messagesTypes: [],
+        configurables: [],
+      },
+    ];
+
+    this.emit(this.events.abis, abis);
+    // #endregion fuel-connector-events-abis
+  }
+}
+
+/**
+ * @group node
+ */
+describe('connectors', () => {
+  describe('instantiation', () => {
+    test('should be able to instantiate a Fuel SDK', () => {
+      // #region fuel-instantiation-1
+      // #import { Fuel };
+
+      const sdk = new Fuel();
```

Comment:
@petertonysmith94 Maybe it makes more sense to call the connector manager variable `manager` or even `connectorManager` on the code snippets.

Reply:
TS SDK and Connector Manger provides a better distinction, thanks for your suggestion :)

45c4f54ee6055e5a1aae0d537f0fc69636bf3419